packages/web-components/src/components/ic-badge/ic-badge.stories.mdx

```diff
@@ -1004,3 +1004,78 @@ import readme from "./readme.md";
     </div> `}
   </Story>
 </Canvas>
+
+### Playground
+
+export const defaultArgs = {
+  accessibleLabel: "notification",
+  customColor: null,
+  maxNumber: 1000,
+  position: "far",
+  size: "default",
+  textLabel: "1",
+  type: "text",
+  variant: "neutral",
+};
+
+<Canvas>
+  <Story
+    args={defaultArgs}
+    parameters={{ loki: { skip: true } }}
+    argTypes={{
+      customColor: {
+        control: { type: "text" },
+      },
+      position: {
+        options: ["far", "near", "inline"],
+        control: {
+          type: "radio",
+        },
+      },
+      size: {
+        options: ["small", "default", "large"],
+        control: {
+          type: "radio",
+        },
+      },
+      type: {
+        options: ["text", "dot", "icon"],
+        control: {
+          type: "radio",
+        },
+      },
+      variant: {
+        options: [
+          "neutral",
+          "success",
+          "warning",
+          "error",
+          "info",
+          "light",
+          "custom",
+        ],
+        control: {
+          type: "select",
+        },
+      },
+    }}
+    name="Playground"
+  >
+    {(args) =>
+      html`<ic-button variant="secondary">
+        <ic-badge
+          slot="badge"
+          accessible-label=${args.accessibleLabel}
+          custom-color=${args.customColor}
+          max-number=${args.maxNumber}
+          position=${args.position}
```

Comment:
Inline position means different things in React vs. Web components

### Web components
<img width="116" alt="Screenshot 2024-06-19 at 09 19 39" src="https://github.com/mi6/ic-ui-kit/assets/113986285/62e0bb99-880f-4b63-9a9a-dba8559405ff">

### React
<img width="141" alt="Screenshot 2024-06-19 at 09 19 56" src="https://github.com/mi6/ic-ui-kit/assets/113986285/2a3d1444-97ac-41e8-9761-69a7c40a712f">

Reply:
Inline position should only really be used with components like tabs, but I couldn't conditionally render a tab/button depending on the position prop, so I think this might have to be left as it is